scl/Makefile.am

```diff
@@ -3,6 +3,7 @@ SCL_SUBDIRS	= \
 	cim		\
 	cisco		\
 	collectd \
+	checkpoint	\
```

Comment:
Please update syslog-ng-core.install too.

Reply:
Please, also include in scl/CMakeLists.txt